docs/pages/develop/development-builds/introduction.mdx

```diff
@@ -1,88 +1,78 @@
-  description="An overview of the development process of building an Expo app to help build a mental model of the core development loop. See where development builds fit in."
-  href="/workflow/overview/"
-  Icon={BookOpen02Icon}
-/>
+## What can't you do in Expo Go and why
 
-<BoxLink
-  title="Getting the most out of Expo Development Builds"
-  description="A YouTube video that explains what a development build is, why it is important, and how to use it."
-  href="https://www.youtube.com/watch?v=7J8LRpja9_o"
-  Icon={VideoRecorderIcon}
+Migrating from Expo Go to a Development Build can be a bit overwhelming the first time. Expo Go is the perfect tool for learning, prototyping and experimenting, but most production apps will convert to using Development builds sooner rather than later. It helps to know exactly what is _not_ possible in Expo Go and _why_, so you can make an informed decision on when and why to make this move.
+
+<VideoBoxLink
+  videoId="FdjczjkwQKE"
+  title="Expo Go & Development Builds: which should you use?"
+  description="In this tutorial video Beto explains what each of them is, and when to choose a Development Build."
 />
 
-## Frequently asked questions
+### Use libraries with native code that aren't in Expo Go
 
-<Collapsible summary="What is a native runtime?">
+Take [react-native-webview](https://docs.expo.dev/versions/latest/sdk/webview/) for example. This library contains native code, but it [is included in Expo Go](https://github.com/expo/expo/blob/main/apps/expo-go/package.json#L23). So when you `npx expo install react-native-webview` on your project, it will install the library in your `node_modules` which includes both the JS code and the native code. But the JS bundle you are building _only_ uses the JS code. Then your JS bundle gets uploaded to Expo Go, and it interacts with the native code that was already bundled with the app.
 
-**A native runtime** refers to the runtime environment that your JavaScript application code is executed in. If your development build is compiled with `expo-camera` installed, the native runtime will include the appropriate code so that you can access that functionality from JavaScript. If the build was not compiled with `expo-camera`, then you cannot access that code from your JavaScript code. Different versions of your app runtime are managed with a [runtime version](/eas-update/runtime-versions/) configuration field.
+If instead you tried to use a library that is not included, e.g. [react-native-firebase](https://docs.expo.dev/guides/using-firebase/#using-react-native-firebase), then you can use the JS code and hot reload the new bundle into Expo Go but it will immediately error because the JS code is trying to call into the native code from the React Native Firebase package that does not exist in Expo Go. There is no way to get the native code into the Expo Go app unless it was already included in the bundle that was uploaded to the App / Play Store.
```

Comment:
```suggestion
Instead, when you try to use a library that is not included, for example, [`react-native-firebase`](/guides/using-firebase/#using-react-native-firebase), then you can use the JS code and hot reload the new bundle into Expo Go but it will immediately error because the JS code tries to call the native code from the React Native Firebase package that does not exist in Expo Go. There is no way to get the native code into the Expo Go app unless it was already included in the bundle that was uploaded to the app stores.
```